server/__test__/service/PostReview.test.js

```diff
@@ -0,0 +1,171 @@
+const reviewService = require('../../src/service/ReviewsService');
+const reviewsDao = require('../../src/repository/ReviewsDAO');
+
+describe('Test to see if Review is Posted', () => {
+    let validateFieldSpy;
+    let postReviewSpy;
+
+    beforeEach(() => {
+        validateFieldSpy = jest.spyOn(reviewService, 'validateFields');
+        postReviewSpy = jest.spyOn(reviewsDao, 'postReview');
+    })
+
+    afterEach(() => {
+        validateFieldSpy.mockRestore();
+        postReviewSpy.mockRestore();
+    })
+
+    /**
+     * Tests for createNewReview function
+     */
+
+    //test to see success of the 
+    test('Successfully Posted Review', async() => {
+        //test object
+        const testSuccess = {
+            username: "John",
+            imageUrl: "Doe.png",
+            rating: "5",
+            content: "Test Success"
+        }
+
+        validateFieldSpy.mockReturnValueOnce(true);
+        postReviewSpy.mockReturnValueOnce(true);
+
+        const result = await reviewService.createNewReview(testSuccess);
+
+        expect(typeof result.recipeId).toBe("string");
+        expect(typeof result.reviewId).toBe("string");
+        expect(result.author).toBe(testSuccess.username);
+        expect(result.imageUrl).toBe(testSuccess.imageUrl);
+        expect(result.rating).toBe(testSuccess.rating);
+        expect(result.content).toBe(testSuccess.content);
```

Comment:
Since `testSuccess` is mutable, you shouldn't check the result against the input.
Within `createNewReview`, the code could have accidentally changed the input parameter `testSuccess`, making checks like these to always be `true` / valid.